openstudiocore/src/model/Space.cpp

```diff
@@ -734,6 +736,22 @@ namespace detail {
     OS_ASSERT(result);
   }
```

Comment:
Check if we need to handle unit in clone or remove, just copy what is done for building story